egs/jesc/mt1/local/score_bleu.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```

Comment:
Can you add some comments on how you change from the original `score_bleu.sh`? This would help the other people to work on multi-byte chars.

Reply:
Comment added. Could you please check it?